src/login.ts

```diff
@@ -62,7 +63,7 @@ export class Login {
 
       try {
         if (previousEntry && previousEntry.password) await this.logout(previousEntry.password)
-      } catch (error) {
+      } catch (error: any) {
```

Comment:
Instead of using `any` on these, could we try something like Kent Dodds is suggesting in [this article](https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript)?

I tried just the simple method (without abstracting out into separate functions) and it worked really well.
```suggestion
      } catch (error) {
      let message
      if (error instanceof Error) message = error.message
      else message = String(error)
      ux.warn(message)
```